src/masoniteorm/models/Model.py

```diff
@@ -144,6 +144,8 @@ class Model(TimeStampsMixin, ObservesEvents, metaclass=ModelMeta):
         "count",
         "decrement",
         "delete",
+        "doesnt_exists",
```

Comment:
This method should be `doesnt_exist` i think and not `doesnt_exists`. the plural on the doesnt_exists doesn't really make gramatically sense.

Reply:
Nevermind this is just a typo here